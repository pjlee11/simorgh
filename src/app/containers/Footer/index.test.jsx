import React from 'react';
import FooterContainer from './index';

import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';

describe(`FooterContainer`, () => {
  shouldShallowMatchSnapshot('should render correctly', <FooterContainer />);
});
