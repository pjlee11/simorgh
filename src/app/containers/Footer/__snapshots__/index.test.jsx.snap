// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterContainer should render correctly 1`] = `
<Footer
  copyrightText="Â© 2018 BBC. The BBC is not responsible for the content of external sites. "
  externalLink={
    Object {
      "href": "https://www.bbc.co.uk/help/web/links/",
      "text": "Read about our approach to external linking.",
    }
  }
  links={
    Array [
      Object {
        "href": "https://www.bbc.com/news/help-41670342",
        "text": "Why you can trust the BBC",
      },
      Object {
        "href": "https://www.bbc.com/terms",
        "text": "Terms of Use",
      },
      Object {
        "href": "https://www.bbc.co.uk/aboutthebbc/",
        "text": "About the BBC",
      },
      Object {
        "href": "https://www.bbc.com/privacy/",
        "text": "Privacy Policy",
      },
      Object {
        "href": "https://www.bbc.com/usingthebbc/cookies/",
        "text": "Cookies",
      },
      Object {
        "href": "https://www.bbc.com/accessibility/",
        "text": "Accessibility Help",
      },
      Object {
        "href": "https://www.bbc.com/contact/",
        "text": "Contact the BBC",
      },
    ]
  }
/>
`;
